packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -317,6 +324,7 @@ abstract class ScrollView extends StatelessWidget {
       cacheExtent: cacheExtent,
       center: center,
       anchor: anchor,
+      clipBehavior: clipBehavior,
```

Comment:
Is `ShrinkWrappingViewport` need `clipBehavior`?

Reply:
`ShrinkWrappingViewport` seems to already have `clipBehavior`: https://github.com/flutter/flutter/blob/77b4505c8014330fba2f2706c658a64bc5d1891b/packages/flutter/lib/src/widgets/viewport.dart#L313

Reply:
I mean line 314. :)

Reply:
Ah yes, nice catch! Fixed now.